### PR TITLE
Logging out from Azure in powershell based tasks

### PR DESCRIPTION
**Pull request by rajatagrawal-dev**

### CODE REVIEW

File: Tasks/Common/VstsAzureHelpers_/Utility.ps1 [hunk 1]
```diff
@@ -484,3 +484,34 @@ function Add-AzureStackAzureRmEnvironment {
...
+            Logout-AzureRmAccount
+        }
+
+        if (Get-Command -Name "Clear-AzureRmContext" -ErrorAction "SilentlyContinue") {
+            Write-Host "##[command]Clear-AzureRmContext -Scope Process"
+            $null = Clear-AzureRmContext -Scope Process
+            Write-Host "##[command]Clear-AzureRmContext -Scope CurrentUser -Force -ErrorAction SilentlyContinue"
+            $null = Clear-AzureRmContext -Scope CurrentUser -Force -ErrorAction SilentlyContinue
+        }
+    } catch {
```
Comment (arjgupta) [inline Tasks/Common/VstsAzureHelpers_/Utility.ps1]: Log the error message as a warning here. "##vso[task.logissue type=warning;]$message"

File: Tasks/Common/VstsAzureHelpers_/Utility.ps1 [hunk 2]
```diff
@@ -484,3 +484,34 @@ function Add-AzureStackAzureRmEnvironment {
...
+            Disconnect-AzureRmAccount
+        }
+        elseif (Get-Command -Name "Remove-AzureRmAccount" -ErrorAction "SilentlyContinue") {
+            Write-Host "##[command]Remove-AzureRmAccount"
+            Remove-AzureRmAccount
+        }
+        elseif (Get-Command -Name "Logout-AzureRmAccount" -ErrorAction "SilentlyContinue") {
+            Write-Host "##[command]Logout-AzureRmAccount"
+            Logout-AzureRmAccount
+        }
```
Comment (arjgupta) [inline Tasks/Common/VstsAzureHelpers_/Utility.ps1]: log a warning here as well saying unable find command to logout session

Comment (rajatagrawal-dev) [reply Tasks/Common/VstsAzureHelpers_/Utility.ps1]: Logged

File: Tasks/Common/VstsAzureHelpers_/Utility.ps1 (hunk 1 above)
Comment (rajatagrawal-dev) [reply Tasks/Common/VstsAzureHelpers_/Utility.ps1]: Logged

**Pull request merged**